modules/mag-deck/mag-deck-arduino/mag-deck-arduino.ino

```diff
@@ -316,6 +323,7 @@ void loop() {
           gcode.print_device_info(device_serial, device_model, device_version);
           break;
         case GCODE_DFU:
+          gcode.send_ack(); // Send ack here since we not reaching the end of the loop
```

Comment:
good call! we should add this to the temp-deck as well

Reply:
Done